docs/dev/code_reference/nist.md

```diff
@@ -0,0 +1,18 @@
+# NIST URLs
+
+::: netutils.nist
+    options:
+        show_submodules: True
+
+**Notes:** 
```

Comment:
thinking about it, this should be in the user guide. This will show up at the bottom of the code, I don't think many would look there.

Reply:
I've added some docs to the user guide that will hopefully work.